src/main/java/core/basesyntax/serviceimpl/ReportWriter.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.serviceimpl;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriter {
+    public void writeReportInFile(String toFileName, String report) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write info" + toFileName, e);
+        }
+    }
```

Comment:
bad method and class name, is it possible to write only report using this class?